AFQ/api.py

```diff
@@ -1353,11 +1378,32 @@ def _tract_profiles(self, row):
                 for ii, scalar in enumerate(self.scalars):
                     scalar_file = self.scalar_dict[scalar](self, row)
                     scalar_data = nib.load(scalar_file).get_fdata()
+                    if isinstance(self.profile_weights, str):
+                        if self.profile_weights == "gauss":
+                            this_prof_weights = gaussian_weights(this_sl)
+                        elif self.profile_weights == "median":
+                            # weights bundle to only return the mean
+                            def _median_weight(bundle):
+                                fgarray = set_number_of_points(bundle, 100)
+                                values = np.array(
+                                    values_from_volume(
+                                        scalar_data,
+                                        fgarray,
+                                        row["dwi_affine"]))
+                                weights = np.zeros(values.shape)
+                                for ii, jj in enumerate(
+                                    np.argsort(values, axis=0)[
+                                        len(values) // 2, :]):
+                                    weights[jj, ii] = 1
```

Comment:
This also seems like a potentially useful way to find this: https://stackoverflow.com/a/64015677/3532933